src/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './App.css';
+
+function App() {
+  return (
```

Comment:
- [ ] Please create a BookList component that renders a list of Books using the Book component.